### DISCUSSION TITLE
Hacktoberfest participation

### DISCUSSION THREAD

**Discussion by shubhamchasing** [open-sauced/app#4153] [Q&A] [answered]
The recent PR #4152 I raised shows a `not-participating` tag on the [Hacktoberfest](https://hacktoberfest.com/).

Although related issue #4126 to the above PR has a `hacktoberfest` label on it, I am not sure why Hacktoberfest is not tracking it.

Can anyone please help me understand this?

![Screenshot from 2024-10-04 13-26-11](https://github.com/user-attachments/assets/2f36f536-fb29-41cd-bc88-89d1b7d26cec)

PR/MR details link: https://hacktoberfest.com/participation/#pr-mr-details

**Comment by jpmcb** [comment]
Hi @shubhamchasing - thanks for raising an issue on this: I see @nickytonline added the `hacktoberfest` label on the issue. I think to have it accepted, we'd have to add some other metadata to the repo and pull request. We hadn't officially discussed participating in Hacktoberfest, but we can probably get that arranged for you.

**Reply by shubhamchasing** [reply]
This is so kind of you, thank you so much for your help.🙏🙌
If I can help in anyway, please let me know.

**Reply by shubhamchasing** [reply]
@jpmcb tagging you in case you missed it

**Reply by nickytonline** [reply]
I remember what the issue is. We have to label the PR with the `hacktoberfest` label once the PR has two approvals. I'm back tomorrow from vacation so will be reviewing more of these tomorrow.

**Reply by shubhamchasing** [reply]
Sorry to disturb you on your vacation, thank you.

**Reply by nickytonline** [reply]
No worries just got back and just happen to see this. 😎

**Reply by shubhamchasing** [reply]
Hey @nickytonline thanks for adding `hacktoberfest` label on PR but this instruction from Hacktoberfest site says either add `hacktoberfest-accepeted` label on PR or add `hacktoberfest` topic to the repository.
https://hacktoberfest.com/participation/#pr-mr-details

**Reply by nickytonline** [reply]
Cool cool. We won't be adding the topic to the repo, so I'll update the label on the PRs.

**Reply by shubhamchasing** [reply]
Thanks and sorry it was a typo from my side it should be `hacktoberfest-accepted`.

**Reply by nickytonline** [reply, marked as answer]
Fixed it. 🚀
